### PR TITLE
The operational transformation library and tool.

### PR DESCRIPTION
**Pull request by dkorolev**
Clean pull request, with no changes to `port.h`, and extra libraries installed on Travis instead. Fingers crossed.

### CODE REVIEW

**Comment by dkorolev** [comment]
@mzhurovich: If this commit turns green -- which I'm hoping for -- let's declare this pull request The Implementation of the OT to go with.

Bedtime for me. Have a great day!
Dima

**Comment by dkorolev** [comment]
@mzhurovich -- this one should be good for a final look-through and merge.

File: Utils/OperationalTransformation/history.cc [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+*******************************************************************************/
+
+#include "ot.h"
+
+#include "../../Bricks/dflags/dflags.h"
+#include "../../Bricks/file/file.h"
+
+DEFINE_string(input, "input.ot", "The name of the input file containing the Operational Transformation of the pad.");
+DEFINE_double(at_m, 0, "If nonzero, the minute at which the contents of the pad should be output.");
+DEFINE_double(interval_s, 5, "If `--at_m` is not set, the iterval between snapshots to be output, in seconds.");
```
Comment (mzhurovich) [inline Utils/OperationalTransformation/history.cc]: s/iterval/interval/

Comment (dkorolev) [inline Utils/OperationalTransformation/history.cc]: :+1: 4d41a23

**Comment by dkorolev** [comment]
@mzhurovich Thx && done && PTAL.

**Pull request merged**